### DISCUSSION TITLE
Rustdesk Client For lubuntu 24.04

### DISCUSSION THREAD

**Discussion by munagala05** [rustdesk/rustdesk#8103] [General]
Dear Sir,
I am instaled the Rustdesk client 1.2.3-2 on Lubuntu 24.04 it will connect from other system and its working fine. But after restart the system it is not connecting from other system. Please Do need Full.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Comment by munagala05** [comment]
OK Thanks.
